### PR TITLE
[RFC] Updated revcheck

### PR DESCRIPTION
**Pull request by saundefined**
- Not all files should be translated, added exceptions that were applied in the [edit.php.net](https://github.com/php/web-doc-editor/blob/master/php/RepositoryManager.php#L1685). 
But I'm not sure that these rules are still actual 🤔 

- In the not translated section added the link to GitHub, where is English content for this file.

- git.php.net link on certain file revision replaced with GitHub link.

### CODE REVIEW

File: scripts/revcheck.php [hunk 1]
```diff
@@ -190,6 +190,39 @@ function populateFileTreeRecurse( $lang , $path , & $output )
...
+            $ignoredDirectories = [
+                'chmonly',
+                'internals',
+                'internals2',
+            ];
+            
+            if(
+                in_array($trimPath, $ignoredDirectories, true) 
+                || in_array($filename, $ignoredFileNames, true)
+                || (strpos($filename, 'entities.') === 0)
```
Comment (kamil-tekiela) [inline scripts/revcheck.php:218]: Are we not running PHP 8 yet? 
Also, the last two `in_array` are missing the strict flag

Comment (saundefined) [reply scripts/revcheck.php:218]: Thanks! Added strict flag, but I'm not sure about PHP 8.

Comment (cmb69) [reply scripts/revcheck.php:218]: This script is supposed to be run locally, AFAIK. Raising requirements to PHP 8.0 might be a bit early; at least that should be discussed on the docs ML.

File: scripts/revcheck.php [hunk 2]
```diff
@@ -190,6 +190,39 @@ function populateFileTreeRecurse( $lang , $path , & $output )
...
+                'README',
+                'DO_NOT_TRANSLATE',
+                'rsusi.txt',
+                'missing-ids.xml',
+            ];
+            
+            $ignoredDirectories = [
+                'chmonly',
+                'internals',
+                'internals2',
```
Comment (Girgias) [inline scripts/revcheck.php:212]: Didn't we remove those docs from EN?

Comment (cmb69) [reply scripts/revcheck.php]: Yes. These should be removed from translations as well.

Comment (saundefined) [reply scripts/revcheck.php]: Looked at the language repositories - both folders are deleted in all translations.
It seems possible to remove

**Comment by saundefined** [comment]
Applied with 010a8aa3015f7dcd5203f7c1e49b74d589d3aeb9 and 74194ed130e3e0e822ff8a8f10f2c5391341e748.
Thanks, everyone!

**Pull request closed without merging**